config.go

```diff
@@ -119,6 +136,33 @@ func init() {
 	}
 }
 
+func parseMatchList(text string) ([]*dto.MetricFamily, error) {
```

Comment:
I've tried this, but it creates panic.

Reply:
Apologize, I had a typo. `matchRules` must be passed by pointer instead of by value.

```diff
- err := yaml.Unmarshal([]byte(text), matchRules)
+ err := yaml.Unmarshal([]byte(text), &matchRules)
```

Reply:
Great! I have fixed it. PTAL. Thanks.